Countries Info/ViewControllers/CountriesListViewController.swift

```diff
@@ -10,41 +10,75 @@ import UIKit
 class CountriesListViewController: UIViewController {
     
     @IBOutlet weak var countriesCollectionView: UICollectionView!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var topView: UIView!
     
     var viewModel: CountriesListViewModelProtocol! {
         didSet {
             viewModel.fetchResult {
-                //reloadData()
+                DispatchQueue.main.async {
+                    self.countriesCollectionView.reloadData()
+                    self.activityIndicator.stopAnimating()
+                }
             }
         }
     }
     
     private let itemPerRow: CGFloat = 1
-    private let sectionInserts = UIEdgeInsets(top: 20, left: 20, bottom: 20, right: 20)
+    private let sectionInserts = UIEdgeInsets(top: 20, left: 13, bottom: 20, right: 13)
```

Comment:
put this value into some constants, like you can create structs **Constants** and put there static let like **topSectionInset**.